test/functional/feature_taproot.py

```diff
@@ -1230,6 +1230,41 @@ def spenders_taproot_nonstandard():
 
     return spenders
 
+def sample_spenders():
```

Comment:
This is helpful for understanding how to write these tests. I might make sense to put this as the first testcase just below  `# === Actual test cases ===`. I worry people might not scroll down the 600 lines to find this. It is a good introduction to the rest of the tests

Reply:
Seems like a matter of taste; leaving as-is for now?

Reply:
Though a practical suggestion but I think it can be left as-is. It's highly likely that the reader would go through the `run_test()` from where they can navigate to the implementation of `sample_spenders()`.